ChromeSeleniumTest.py

```diff
@@ -0,0 +1,11 @@
+import time
```

Comment:
Is this just for testing? It might be something better off removed than checked in if we're not expecting it to keep adding value going forward